extensions/parquet/base/src/main/java/io/deephaven/parquet/base/ColumnWriterImpl.java

```diff
@@ -74,6 +79,12 @@ public class ColumnWriterImpl implements ColumnWriter {
             final int targetPageSize,
             final ByteBufferAllocator allocator) {
         this.writeChannel = writeChannel;
+        // The buffered output stream is used to buffer the page header. Therefore, we set the size of the buffer as
+        // 16 KB because that is the expected page header size (ref
+        // https://github.com/apache/arrow/blob/ac581fd2a87b35c872cf334bb147851fe1287714/cpp/src/parquet/column_reader.h#L57)
```

Comment:
It seems like this is the default unit of growth for page size buffers in the arrow reading code. I would rather not reference another project from our code, unless it's Parquet or Java, etc. If you think our usual size is much smaller, we might want to default to something smaller, but I'm not uncomfortable with 16KB.

Reply:
Leaving the buffer size as 16 KB just to be safe side.